### ISSUE TITLE
Website won't load

### ISSUE THREAD

**Issue by AmyOlex** [jminnier/STARTapp#16]
Hello,

This is a great tool, and I use it in an assignment for a bioinformatics class I teach; however, this past week several student and myself are unable to load the website from https://kcvi.shinyapps.io/START/.  It just hangs and says "Please Wait".

**Comment by jminnier** [comment]
Hi @AmyOlex, thank you so much for bringing this to my attention. It's very strange it just suddenly stopped working, but I've gone in and restarted the app on shinyapps and it seems to be working again now! Sorry for the trouble, and I hope it didn't derail your assignment too much. It's great to hear you are using it for edcuation, let me know if you ever have feedback about how to make it better for your class, too!

**Comment by AmyOlex** [comment]
@jminnier Thank you!  It is working again.  Only a few students had an issue, so hoping it is resolved now.  I gave them instructions to get the app running on their laptops, but the web service is prefered as debugging install issues can be a problem when teaching remotely!  This tool is great for introducing gene expression analysis in a hassle free way.  The only thing I did was preformat some data from GEO for the students.  Importing data directly from GEO without having to format it manually is the only improvement I can think of at the moment.  Otherwise it is a very nice tool!  Thank you for sharing it!
